.pre-commit-config.yaml

```diff
@@ -19,6 +19,11 @@ repos:
 #    hooks:
 #      - id: pyupgrade
 #        entry: pyupgrade --py38-plus
+-   repo: https://github.com/MarcoGorelli/polars-upgrade
```

Comment:
I don't think we want this hook here as upgrading `polars` syntax will often break backwards compatibility. Users with different `polars` versions will start getting errors.

Reply:
This I understand, noted.